src/components/Book.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Book = () => {
+  const books = [
+    { id: 1, name: 'book1' },
+    { id: 2, name: 'book2' },
+  ];
+
+  return books.map((book) => (
+    <li key={book.id}>
+      {book.name}
+      <button type="button">remove</button>
+    </li>
+  ));
```

Comment:
- [ ] This function should have a propTypes and  validate it.

![lmlml](https://user-images.githubusercontent.com/61562626/132662373-eff2f57b-6fb8-4f0e-84ca-30f9980cccd6.jpeg)